components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add-finish-ajaxprocessor.jsp

```diff
@@ -108,8 +109,10 @@
             app.setIdTokenExpiryTime(Long.parseLong(idTokenExpiryTime));
             app.setTokenType(tokenType);
 
-            if (Boolean.parseBoolean(request.getParameter("enableBackchannelLogout"))) {
-                app.setBackChannelLogoutUrl(backchannelLogoutUrl);
+            if (OAuthConstants.OIDCConfigProperties.BACK_CHANNEL_LOGOUT_SELECTED.equalsIgnoreCase(logoutMechanism)) {
+                app.setBackChannelLogoutUrl(logoutUrl);
+            } else if (OAuthConstants.OIDCConfigProperties.FRONT_CHANNEL_LOGOUT_SELECTED.equalsIgnoreCase(logoutMechanism)) {
```

Comment:
can you check whether the line length is within 120

Reply:
Fixed in commit: PR issues resolved.